floyd/cli/experiment.py

```diff
@@ -171,6 +187,12 @@ def output(id, url, download):
     if "output" in task_instance.output_ids:
         output_dir_url = "{}/api/v1/resources/{}?content=true".format(floyd.floyd_host,
                                                                       task_instance.output_ids["output"])
+        if format == 'json':
+            experiment_dict = json.loads(experiment.to_dict())
```

Comment:
it's doing double encoding/decoding here. a better way to do it is to get the dictionary, add the output_dir_url key, then serialize to json string at the end.